Source/Scene/ClippingPlane.js

```diff
@@ -25,6 +25,9 @@ define([
      * opposite to the normal; if zero, the plane passes through the origin.
      */
     function ClippingPlane(normal, distance) {
+        Check.typeOf.object('normal', normal);
+        Check.typeOf.number('distance', distance);
```

Comment:
Wrap these 2 lines in

        //>>includeStart('debug', pragmas.debug);
        //>>includeEnd('debug');